Makefile

```diff
@@ -257,6 +257,8 @@ bundle-base:
 	cp ${OUTPUT_IMAGE_TAR_DIR}/astra-connector-images.tar $(INSTALL_BUNDLE_DIR)
 	cp ${MAKEFILE_DIR}/controllerconfig.yaml $(INSTALL_BUNDLE_DIR)/controllerconfig.yaml
 	cp ${MAKEFILE_DIR}/astraconnector_operator.yaml $(INSTALL_BUNDLE_DIR)/astraconnector_operator.yaml
+	cp ${MAKEFILE_DIR}/config/samples/astraconnector_v1.yaml $(INSTALL_BUNDLE_DIR)/astraconnector_v1.yaml
```

Comment:
I guess I am confused of why we are doing this?  
There is a astraconnector.yaml in the root directory that is updated with the right version of the connector with each build.
What is this going to do?  And does it get the replacement of the image tag?

Reply:
It allows our downstream CICD pipelines to have one bundle to pull, instead of having to make a series of requests to grab all of the files required to install the connector.
It will also ensure that you are getting the correct version of the operator yaml.

Reply:
Oh I'm just noticing the controllerconfig file already being copied in here. Ok I'll delete this line